language/move-symbol-pool/src/pool.rs

```diff
@@ -69,7 +69,7 @@ impl Pool {
     /// which top-level bucket contains an entry corresponding to the string,
     /// as well as a scalar value that can be used to quickly check whether
     /// two strings are not equal.
-    fn hash(string: &str) -> u64 {
+    fn hash(string: &Cow<str>) -> u64 {
```

Comment:
Just to make sure I understand, this is to avoid a clone that occurs when creating a hash for the `insert` method? I guess it could be split out into its own pull request, but whatever (GitHub makes it so annoying to split things up and stack them...).

Reply:
There shouldn't be a change here. This was in another PR that I thought had already landed 

> GitHub makes it so annoying to split things up and stack them...
lol it is why my PRs end up being so big a lot of times :P